lib/nostrum/voice/opus.ex

```diff
@@ -147,35 +130,37 @@ defmodule Nostrum.Voice.Opus do
         } = _state
       )
       when is_list(body) do
+    segment_table_raw =
+      body
+      |> Enum.map(&byte_size(&1))
+      |> encode_ogg_segment_table()
+
     %{
       version: 0,
       header_type: header_type,
       granule_position: granule_position,
       bitstream_serial: bitstream_serial,
       page_sequence: page_sequence,
       crc_checksum: 0,
-      page_segments: body |> Enum.count(),
-      segment_table_raw:
-        body
-        |> Enum.map(&byte_size(&1))
-        |> encode_ogg_segment_table(),
+      page_segments: segment_table_raw |> byte_size(),
```

Comment:
Personal preference, feel free to ignore:
```suggestion
      page_segments: byte_size(segment_table_raw),
```